lib/menu.ts

```diff
@@ -412,7 +412,13 @@ export function menuItemSubmenu(
       (yesNo: boolean) => yesNo || ctl.close()),
 
     // Clicks that open a submenu should not cause parent menu to close.
-    dom.on('click', (ev) => { ev.stopPropagation(); }),
+    dom.on('click', (ev, elem) => {
+      if (options.action && !elem.classList.contains('disabled')) {
+        options.action(ev.target as HTMLElement);
```

Comment:
What do you think about also passing in `ev` here? Might save us from a future PR if we need to keep the parent menu open when `action` is called.